### ISSUE TITLE
Modal data

### ISSUE THREAD

**Issue by yuritoledo** [taylorsam209/calendar-scheduler#1]
Hey man

Is possible to change the fields inside modal? (after clicking in a day)
Maybe add more text fields or checkboxes, etc.

**Comment by taylor-zennify** [comment]
Yes its possible to change and add more. Take a look at Calendar.js and see the methods for description and title. Look at the state for those. If you want to add another field, try to duplicate those methods with your specific fields you want.
